### PR TITLE
CI: update poplar sdk version

### PR DESCRIPTION
**Pull request by kragniz**
## What does this PR do?

Bumps the poplar SDK version so we can upgrade the required pytorch version.

## Before submitting

- [ ] Was this **discussed/approved** via a GitHub issue? (not for typos and docs)
- [x] Did you read the [contributor guideline](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/.github/CONTRIBUTING.md), **Pull Request** section?
- [x] Did you make sure your **PR does only one thing**, instead of bundling different changes together?
- [ ] Did you make sure to **update the documentation** with your changes? (if necessary)
- [ ] Did you write any **new necessary tests**? (not for typos and docs)
- [ ] Did you verify new and **existing tests pass** locally with your changes?
- [ ] Did you list all the **breaking changes** introduced by this pull request?
- [ ] Did you **update the [CHANGELOG](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/CHANGELOG.md)**? (not for typos, docs, test updates, or internal minor changes/refactorings)

## PR review

Anyone in the community is welcome to review the PR.
Before you start reviewing make sure you have read [Review guidelines](https://github.com/PyTorchLightning/pytorch-lightning/wiki/Review-guidelines). In short, see the following bullet-list:

- [x] Is this pull request ready for review? (if not, please submit in draft mode)
- [ ] Check that all items from **Before submitting** are resolved
- [ ] Make sure the title is self-explanatory and the description concisely explains the PR
- [ ] Add labels and milestones (and optionally projects) to the PR so it can be classified

## Did you have fun?

Make sure you had fun coding 🙃

### CODE REVIEW

**Comment by krshrimali** [comment]
Hi, @kragniz - Thank you so much for updating the version. Should we also bump the PyTorch version if the SDK requires PyTorch 1.9 version? https://github.com/PyTorchLightning/pytorch-lightning/blob/master/.github/workflows/ci_dockers.yml#L129

**Review by Borda — APPROVED** [review]

**Review by justusschock — APPROVED** [review]

File: tests/accelerators/test_ipu.py [hunk 1]
```diff
@@ -343,7 +343,7 @@ def test_autoreport(tmpdir):
     )
     trainer.fit(model)
     assert os.path.exists(autoreport_path)
-    assert os.path.isfile(autoreport_path + "profile.pop")
+    assert os.path.isfile(autoreport_path + "training/profile.pop")
```
Comment (kaushikb11) [inline tests/accelerators/test_ipu.py:346]: Could you mention to the reviewers why this change is required?

**Review by kaushikb11 — APPROVED** [review]

File: tests/accelerators/test_ipu.py (hunk 1 above)
Comment (kaushikb11) [reply tests/accelerators/test_ipu.py:346]: And would it affect other parts of the codebase?

**Review by carmocca — APPROVED** [review]

**Review by krshrimali — APPROVED** [review]

File: tests/accelerators/test_ipu.py (hunk 1 above)
Comment (kragniz) [reply tests/accelerators/test_ipu.py:346]: There was a change in 2.2 to allow people to pass a modelName, which is used to place profiles in separate directories. If it's not specified, it defaults to `inference` / `training`.

https://docs.graphcore.ai/projects/poptorch-user-guide/en/latest/reference.html#poptorch.Options.modelName
